src/quo2/components/markdown/text.cljs

```diff
@@ -23,7 +23,8 @@
          {:text-align (or align :auto)}
          (if (:color style)
            style
-           (assoc style :color (if (= (theme/get-theme) :dark) colors/white colors/neutral-100)))))
+           (assoc style :color (or secondary-color
```

Comment:
why can't we just use `{:style {: color "overwrite from here" }}` ??

Reply:
I don't get this comment Jamie, Can you please elaborate ?